src/app/pages/common/entity/entity-wizard/entity-wizard.component.ts

```diff
@@ -108,11 +108,10 @@ export class EntityWizardComponent implements OnInit {
     });
 
     for (const i in this.conf.wizardConfig) {
+      const formGroup = < FormGroup > this.formArray.get(i);
```

Comment:
[Please prefer](https://google.github.io/styleguide/tsguide.html#type-assertions-syntax) `as FormGroup` for casting.